Back end/src/main/java/com/onbank/OnbankApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.onbank;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
```

Comment:
Dożućmy explicite deklarację tego z którego pliku będziemy brać properties (najlepiej niech lokalizację wskacuje się przez parametr jvm). WebMVC też włączymy ręcznie żeby było to oczywiste.

Reply:
@PropertySource("classpath:application.properties") - taką adnotacją deklarujemy jawnie plik properties. Nie wiem jak zrobić to poprzez parametr jvm.
@EnableWebMvc 
Czy te adnotacje użyte nad klasą OnBankApplication będą ok?

Reply:
Na pierwsze pytanie mogę odpowiedzieć :P Można użyć @PropertySource("file:${app.home}/app.properties") do wskazania ścieżki do pliku, a to w środku to zmienna środowiskowa, która zawiera w sobie po prostu ścieżkę do pliku, na przykład na lokalnym komputerze. I tą konfigurację można wrzucić do osobnej klasy konfiguracyjnej AppConfig.